problems/problems.go

```diff
@@ -0,0 +1,74 @@
+package errors
+
+import (
+	"fmt"
+)
+
+//Problem implements the RFC7807 "problem"/error standard
+//Additional error field can be defined in custom struct implementing the Problem struct
+type Problem struct {
```

Comment:
Do we have a better make for this?

Reply:
The name problem refers to the description of the RFC standard "Problem Details".

Nonetheless, we could name it `Error`, as it implements `error` in Go.